knowledge-base/kb-chat-showing-message-desktop-alert.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Show desktop alert with message
```

Comment:
Use RadChat, chat and ConversationalUI in the meta attributes. This way it is more clear what control you are referring to. Usually, this page will be found via Google search so if the name of the control is in the search result, then it is more likely the user to click it.